src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -233,6 +233,10 @@ public function load(array $configs, ContainerBuilder $container)
         }
 
         if ($this->isConfigEnabled($container, $config['session'])) {
+            if (!\extension_loaded('session')) {
+                throw new \LogicException('PHP extension "session" is required.');
```

Comment:
I think this should say "... because the session config is enabled" to make it clear that the alternative might be to disable sessions (which got enabled automatically because of flex).